src/components/modal/Modal.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import Cancel from '../../images/cancel.png'
 import './modal.css'
+import ReactPlayer from 'react-player'
 const Modal = (props) => {
```

Comment:
object destructuring can be done for props here as well

Reply:
Thanks I will fix these...